src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs

```diff
@@ -115,9 +115,10 @@ public string GetCurrentRequestHeader(ApiEvent eventInProgress)
             platformConfig.Append(ConvertFromStringToBitwise(isTokenCacheSerialized) + ",");
             platformConfig.Append(ConvertFromStringToBitwise(isLegacyCacheEnabled) + ",");
             
-            platformConfig.Append(Metrics.TotalDurationInMs + ",");
             platformConfig.Append(Metrics.TotalAccessTokensFromIdP + ",");
-            platformConfig.Append(Metrics.TotalAccessTokensFromCache);
+            platformConfig.Append(Metrics.TotalAccessTokensFromCache + ",");
```

Comment:
I did not understand that we want to send these as telemetry. What is the point? Telemetry string is pretty limited in size, we should be selective with data points.